frontend/components/organiser/forms/FormBackButton.tsx

```
@@ -0,0 +1,21 @@
+import { ArrowLeftIcon } from "@heroicons/react/24/outline";
+
+interface FormBackButtonProps {
+  onClick: (event: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void;
+}
+
+const FormBackButton = ({ onClick }: FormBackButtonProps) => {
+  return (
+    <button
+      onClick={onClick}
+      className="flex items-center justify-center min-w-16 gap-2 px-2.5 py-2.5 bg-white rounded-lg hover:bg-gray-50 transition-colors sm:border sm:border-gray-200"
+    >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)